File: pset6/mario.py
# import modules
import cs50

# ask user for input
while True:
    print("Height: ", end = "")
    height = cs50.get_int()
    if not height < 0 and not height > 23:
        break

# print half-pyramid
for row in range(height):
    print(" " * (height - row - 1), end = "")
    print("#" * (2 + row), end = "")
    print()

File: pset1/mario.c
// include libraries
#include <stdio.h>
#include <cs50.h>

int main(void)
{
  // initialize variables
  int height;
  // ask user for input
  do
  {
    printf("Height: ");
    height = get_int();
  }
  while (height < 0 || height > 23);
  // print spaces and hashes in shape of half-pyramid, using user input for pyramid's height
  for(int row = 1; row <= height; row++)
  {
    for(int space = height - row; space > 0; space--)
    {
      printf(" ");
    }
    for(int hash = height - row; hash <= height; hash++)
    {
      printf("#");
    }
    printf("\n");
  }
}